tests/javascript/sendtestmail/spec-setup.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ * @package     Joomla
+ * @subpackage  JavaScript Tests
+ * @since       3.6
+ * @version     1.0.0
+ */
+
+define(['jquery', 'text!testsRoot/sendtestmail/fixtures/fixture.html', 'libs/sendtestmail', 'libs/core'], function ($, fixture) {
+	$('body').append(fixture);
+
+	mailResponses = {
+		success: {
+			status: 200,
+			statusText: 'HTTP/1.1 200 OK',
+			responseText: '{"messages": {}}'
```

Comment:
As I remember it only checks if the type of message is an object or not. That's why I put it that way. But still, yeah would be nice to have some content. I'll add something :)